### ISSUE TITLE
want "long" mode output

### ISSUE THREAD

**Issue by davepacheco** [oxidecomputer/looker#4]
The default output looks similar to `bunyan -o short`, which is convenient in a lot of cases.  It skips the date part of the timestamp, the hostname, and I'm sure other stuff.  There are other cases where I want full timestamps, hostnames, etc. the way `bunyan` does in its default mode.  It'd be nice to have a `--long` mode for that.

**Comment by jclulow** [comment]
I have added a `-o OUTPUT` option:

```
$ looker --help
...
    -o, --output FORMAT output format:
                        - "short" is the default output format
                        - "long" prints all fields and long timestamps
```

The only completely absent fields were `hostname` and `pid`, which I have added to the `long` output.  The `long` output also displays the date as well as the time:

![newoutput](https://user-images.githubusercontent.com/304070/198854629-56d7bd3c-1aa1-46d4-bb42-bdca077341ac.png)

Fixed in 1eb3b192caaa66d96f5be4471f281121cdbf8fd9.

**Comment by davepacheco** [comment]
Thanks!

Would you be amenable to having it be "2022-10-01T15:01:23.123Z" instead of "2022-10-01 15:01:23.123Z"?  Then it would be properly ISO 8601, which is convenient for pasting into other tools.  (I paste these into https://github.com/davepacheco/dates all the time to see how far apart two timestamps are, or to generate a timestamp that differs from another one by a fixed delta.)
